okonomiyaki/runtimes/runtime_info.py

```diff
@@ -209,7 +209,7 @@ def _compute_variables(metadata, prefix, name):
     variables = dict(
         (k, v) for k, v in data.items() if isinstance(v, six.string_types)
     )
-    variables["prefix"] = prefix
+    variables["prefix"] = prefix.replace('$', '$$')
```

Comment:
I agree with you that replacing `$` with `$$` unconditionaly is not the right way. We can probably use a regex.

Reply:
This is not necessary now that we are requiring curly braces for the substitution.